### DISCUSSION TITLE
V3 jquery.event.drop

### DISCUSSION THREAD

**Discussion by kofifus** [6pac/SlickGrid#708] [Q&A] [answered]
looking at the instructions for new V3 ( https://github.com/6pac/SlickGrid/wiki/Major-version-3.0----Removal-of-jQueryUI-requirement-(replaced-by-SortableJS) )

There is no mention of `jquery.event.drop` - is `jquery.event.drop-2.3.0.js` not needed anymore as well ?

**Comment by ghiscoding** [comment]
I don't think it was ever needed, except for a very special Example [Row selection & reordering](http://6pac.github.io/SlickGrid/examples/example9-row-reordering.html) which only uses `jquery.event.drop` for dropping a row hover the Recycle Bin button.... So in summary, no it's no needed at all (and probably never was)

Actually I did mention about `jquery.event.drop` in the [Quick Summary](https://github.com/6pac/SlickGrid/wiki/Major-version-3.0----Removal-of-jQueryUI-requirement-(replaced-by-SortableJS)#main-changes---quick-summary-list) section on the top but yes I agree it might not be clear that it's not needed. Feel free to propose changes or suggestions

Moving this to a "Discussion" since it's not an issue but more of a question/discussion

**Comment by kofifus** [comment, marked as answer]
got it thx!
